src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -43,9 +43,18 @@ class Players(
         roster.forEach(Player::dealerDealtBlackjack)
     }
 
-    fun result(dealer: Dealer): List<PlayerResult> {
+    fun placeBets(bets: List<Bet>) {
+        requireBetsSizeEqualsRosterSize(bets)
+        roster.zip(bets).forEach { (player, bet) -> player.placeBet(bet) }
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
코틀린 공식 문서에 따르면 함수 체인의 경우 아래와 같이 체인별 개행을 해주어야 합니다.
(일반적으로 체인의 첫 번째 호출에는 개행이 있어야 하지만 개행하지 않는 편이 더 합리적이라면 생략해도 괜찮습니다)

```suggestion
        roster.zip(bets)
            .forEach { (player, bet) -> player.placeBet(bet) }
```